### DISCUSSION TITLE
Scripting Support

### DISCUSSION THREAD

**Discussion by jlorezz** [Eugeny/tabby#6097] [Q&A]
Does tabby support scripting?

**Comment by jlorezz** [comment]
for example opening multiple panes from a script and running commands inside them?

**Reply by Eugeny** [reply]
You can do all these things from a plugin, but there isn't any built-in scripting UI

**Reply by jlorezz** [reply]
what plugin is it?

**Reply by jlorezz** [reply]
I see there is one called workspace manager but not sure if that is accessible from a sh script.

**Reply by williamblair333** [reply]
Anyone know which plugin it is?

**Reply by Eugeny** [reply]
It was tabby-workspace-manager, but Tabby now has native support for login scripts and saving multiple panes as a single profile :)

**Reply by williamblair333** [reply]
My search skills are probably bad - is there an example of how to create and implement a login script?

**Reply by dtap001** [reply]
I would like same feature like termius snippets. Is that possible?

**Reply by ISnotes** [reply]
> I would like same feature like termius snippets. Is that possible?
+1

**Comment by dtap001** [comment]
I came to the conclusion that Espanso is a much better alternative then integrating scripting in tabby.
